Examples/PatchExample.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace QModManager.Examples
+{
+    using QModManager.API.ModLoading;
+
+    [QModCoreInfo("QExampleMod", "Example Mod", "QModManager Dev Team", API.Game.Subnautica)]
+    [QModLoadBefore("SomeMod")]
+    [QModLoadBefore("SomeMod2")]
+    [QModLoadAfter("SomeOtherMod")]
+    [QModDependency("ApiMod")]
+    [QModDependency("OtherAPIMod", 2, 2, 1)]
+    internal static class PatchExample
+    {
+        [QModPatchMethod]
+        public static void PatchMyMod()
+        {
```

Comment:
Idea: Let patch methods return a `State` which can either be `OK`, `Error`, or `Cancel` (names up for debate) that change the way QModManager treats them. For example, if `State.Error` is returned, the mod is marked as errored, and if `State.Cancel` is returned, the mod's loading is cancelled but it's not mark as errored.

Reply:
Not a terrible idea. But we can't enforce anything about method signatures with attributes.  
Are you still sure you want to include this?